src/qualified_name_is_fully_qualified.hack

```diff
@@ -0,0 +1,14 @@
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HHAST;
+
+function qualified_name_is_fully_qualified(QualifiedName $name): bool {
```

Comment:
I think I remember from the original PR, you wanted to add this to the class?

You can do that by adding the class name to `CodegenSyntax::getKindsWithManualSubclasses()`, then `bin/update-codegen` will generate an abstract class that you can extend and add any methods you need.

Reply:
This dev ex improvement had to be postponed. See 6011173 for more details. It would conflict with another linter I wrote later.

Reply:
https://github.com/hhvm/hhast/pull/276#discussion_r393260011